CMakeLists.txt

```diff
@@ -212,7 +212,7 @@ if (MPI)
         endif()
     endif()
 
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${MPI_Fortran_COMPILE_FLAG}")
+    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${MPI_Fortran_COMPILE_FLAG} -fallow-invalid-boz")
```

Comment:
Is it safe to add `-fallow-invalid-boz` by default?
Should it be added under some kind of test like "if COMPILER = GNU and OS = WINDOWS"?
Opened question! Not sure what is best

Reply:
That's a good point, I just followed the compiler complaint without much second thought 😄
Though, what bothers me is why the compiler generated errors for lines which are comments in the first place... I am not familiar with Fortran so hard to say

Reply:
> That's a good point,

OK, add this to the PR, and it's good to go for me.

Reply:
Done. It seems this is related to the code being old Fortran 77